src/config.js

```diff
@@ -7,7 +7,7 @@ var env = 'production',
 // --- Env Specific ---
 if (location.hostname.match(/localhost|(\.(dev|app))$/)) {
     env = 'development';
-    apiBase = 'http://arcv-service.app/api';
+    apiBase = 'http://service-arcv.app/api';
```

Comment:
Where is this set? Is it just your local? I think the arcv-service.app/api was more intuitive (because repo name)